docs/outreach/apply.md

```diff
@@ -92,9 +92,9 @@ As you are getting started on your first pull request:
   [graphical Git client](../git/setup.md#get-a-graphical-client).
 
 - Construct [coherent, mergeable
-  commits](../contributing/version-control.md#commit-discipline), with clear
+  commits](../contributing/commit-discipline.md), with clear
   commit messages that follow the [Zulip commit style
-  guide](../contributing/version-control.md#commit-messages). More broadly, clear
+  guide](../contributing/commit-discipline.md). More broadly, clear
```

Comment:
This translation produces duplicate links; I'm pretty sure the second link should still be to the commit messages anchor.